diff --git a/node_modules/nodejs-whisper/dist/WhisperHelper.js b/node_modules/nodejs-whisper/dist/WhisperHelper.js
index df5fec2..f8d5b33 100644
--- a/node_modules/nodejs-whisper/dist/WhisperHelper.js
+++ b/node_modules/nodejs-whisper/dist/WhisperHelper.js
@@ -24,7 +24,7 @@ const constructCommand = (filePath, args) => {
         throw new Error(errors.join('\n'));
     }
     const modelName = constants_1.MODEL_OBJECT[args.modelName];
-    let command = `./main  ${constructOptionsFlags(args)} -l ${((_a = args.whisperOptions) === null || _a === void 0 ? void 0 : _a.language) ? (_b = args.whisperOptions) === null || _b === void 0 ? void 0 : _b.language : "auto"} -m ./models/${modelName}  -f ${filePath}`;
+    let command = `main  ${constructOptionsFlags(args)} -l ${((_a = args.whisperOptions) === null || _a === void 0 ? void 0 : _a.language) ? (_b = args.whisperOptions) === null || _b === void 0 ? void 0 : _b.language : "auto"} -m models/${modelName}  -f ${filePath}`;
     return command;
 };
 exports.constructCommand = constructCommand;
diff --git a/node_modules/nodejs-whisper/dist/whisper.js b/node_modules/nodejs-whisper/dist/whisper.js
index c47302f..c7792a0 100644
--- a/node_modules/nodejs-whisper/dist/whisper.js
+++ b/node_modules/nodejs-whisper/dist/whisper.js
@@ -16,7 +16,7 @@ exports.executeCppCommand = exports.whisperShell = void 0;
 const path_1 = __importDefault(require("path"));
 const shelljs_1 = __importDefault(require("shelljs"));
 const WHISPER_CPP_PATH = path_1.default.join(__dirname, '..', 'cpp', 'whisper.cpp');
-const WHISPER_CPP_MAIN_PATH = './main';
+const WHISPER_CPP_MAIN_PATH = 'main';
 const projectDir = process.cwd();
 const defaultShellOptions = {
     silent: true,
